app/src/main/java/com/tananaev/passportreader/MainActivity.java

```diff
@@ -404,12 +524,25 @@ protected void onPostExecute(Exception result) {
 
                 MRZInfo mrzInfo = dg1File.getMRZInfo();
 
-                intent.putExtra(ResultActivity.KEY_FIRST_NAME, mrzInfo.getSecondaryIdentifier().replace("<", " "));
-                intent.putExtra(ResultActivity.KEY_LAST_NAME, mrzInfo.getPrimaryIdentifier().replace("<", " "));
-                intent.putExtra(ResultActivity.KEY_GENDER, mrzInfo.getGender().toString());
+                intent.putExtra(ResultActivity.KEY_FIRST_NAME, WordUtils.capitalizeFully(mrzInfo.getSecondaryIdentifier().replace("<", " ")));
+                intent.putExtra(ResultActivity.KEY_LAST_NAME, WordUtils.capitalizeFully(mrzInfo.getPrimaryIdentifier().replace("<", " ")));
+                intent.putExtra(ResultActivity.KEY_GENDER, WordUtils.capitalizeFully(mrzInfo.getGender().toString()));
                 intent.putExtra(ResultActivity.KEY_STATE, mrzInfo.getIssuingState());
                 intent.putExtra(ResultActivity.KEY_NATIONALITY, mrzInfo.getNationality());
 
+                String passiveAuthStr = "";
+                String chipAuthStr = "";
+                if(passiveAuthSuccess)
```

Comment:
Also make sure that there is always a space after "if" and overall styling matches existing code.